### PR TITLE
Arms proposal post

### PR DESCRIPTION
**Pull request by DillonFlohr**
Just a straight copy of the proposal that was written from ARMS. I thought you mentioned that the proposal would be appropriate to have on the site as well so I whipped it up real quick.

### CODE REVIEW

File: _posts/2018-10-15-arms-proposal.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# CSC 596 Proposal
+
+### ARMS (Autonomous Robot Markup Syntax) for ARCS
+
+### Student: Dillon Flohr
+
+### Supervisor: Anthony J. Clark
+
+In robot simulation, there are a variety of formats used to specify the parts, dimensions, and features of a robot. It can be very laborious to repeatedly create these specifications for the same robot for different simulation systems, when the only difference could be the formatting of the XML input. The goal of this self-study course is to create a language and interpreter that will take a text file written in Autonomous Robot Markup Syntax (ARMS), a Light Markup Language we will create, and output various robot specification files in the form of code. These files will be usable in their respective simulation environment and all come from the same source file.
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: there are a variety of formats used --> a variety of formats are used

Comment (anthonyjclark) [reply _posts/2018-10-15-arms-proposal.md]: ~~repeatedly~~

File: _posts/2018-10-15-arms-proposal.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We will be focusing on creating ARMS and getting it working for the Open Dynamics Engine. This alone will take up the entirety of the course work, and any extra time we may have after completing the first format will be used to start the other targets.
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: We will be focusing on creating --> We will focus on creating

File: _posts/2018-10-15-arms-proposal.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+We will be meeting weekly during the first half of the semester on Thursdays from 4:00 to 4:45. For the second half of the semester we will meet at this time bi-weekly. We will also be using the ARCS Slack channel to communicate.
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: We will be meeting weekly during the first half of the semester --> During the first half of the semester, we will meet weekly

File: _posts/2018-10-15-arms-proposal.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+categories: ["Missouri State University", "ARCS"]
+tags: ["self-study", "csc596", "robotics", "python"]
+author: "Dillon Flohr"
+---
+
+# CSC 596 Proposal
+
+### ARMS (Autonomous Robot Markup Syntax) for ARCS
+
+### Student: Dillon Flohr
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: Just make this bold instead of an h3.

File: _posts/2018-10-15-arms-proposal.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+### Supervisor: Anthony J. Clark
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: Just make this bold instead of an h3.

File: _posts/2018-10-15-arms-proposal.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "ARMS CSC 596 Proposal"
+categories: ["Missouri State University", "ARCS"]
+tags: ["self-study", "csc596", "robotics", "python"]
+author: "Dillon Flohr"
+---
+
+# CSC 596 Proposal
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: Combine this with the h3 below. It should be the same as the title in yaml at the top of the file.

# ARMS CSC 596 Proposal

File: _posts/2018-10-15-arms-proposal.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+### ARMS (Autonomous Robot Markup Syntax) for ARCS
```
Comment (anthonyjclark) [inline _posts/2018-10-15-arms-proposal.md]: Remove

File: _posts/2018-10-15-arms-proposal.md (hunk 2 above)
Comment (DillonFlohr) [reply _posts/2018-10-15-arms-proposal.md]: Done

**Pull request merged**